samples/WingTipToys/src/App_Start/RouteConfig.cs

```diff
@@ -8,11 +8,13 @@ namespace WingtipToys
 {
     public static class RouteConfig
     {
-        public static void RegisterRoutes(RouteCollection routes)
+        public static void RegisterRoutes(System.Web.Routing.RouteCollection routes)
         {
             var settings = new FriendlyUrlSettings();
             settings.AutoRedirectMode = RedirectMode.Permanent;
+#if !NET
```

Comment:
I prefer the PORT_ to identify areas needed - or at least a comment for posterity